From: Christopher Obbard <chris.obbard@collabora.com>
Date: Thu, 26 Nov 2020 14:33:49 +0000
Subject: doc/examples: Don't wait for user to enter password

The example user setup script calls adduser which in turn calls passwd
to ask the user for a password; which waits indefinitely for user
input. Since we set the password noninteractively, disable adduser
from asking the user to set a password.

Fixes: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1006823
Signed-off-by: Christopher Obbard <chris.obbard@collabora.com>
(cherry picked from commit c8c5075853aab9e1ac6ae07a3a7c2b070aa38a62)
---
 doc/examples/setup-user.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/examples/setup-user.sh b/doc/examples/setup-user.sh
index da155d1..07b29ee 100755
--- a/doc/examples/setup-user.sh
+++ b/doc/examples/setup-user.sh
@@ -3,7 +3,7 @@
 set -e
 
 echo "I: create user"
-adduser --gecos User user
+adduser --gecos User --disabled-password user
 
 echo "I: set user password"
 echo "user:user" | chpasswd
